src/components/PopUp.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { css, cx } from "emotion";
+import Modal from "./Modal";
+import Button from "./Button";
+import { mixins } from "../theme";
+import * as styles from "./styles/PopUp.styles";
```

Comment:
This is not actually styles but classNames. Returns a string. name might be confusing.